data-types.md

```diff
@@ -0,0 +1,90 @@
+# Data types
+
+There are four main data types that you can use to describe the data in your columns. These four are strings, decimal, float and boolean.
+
```

Comment:
There in "integer" type also, some our csv files have the observations with integer, such as number of companies. Observation is either decimal or integer.